src/index.njk

```diff
@@ -68,6 +64,20 @@ masthead: true
         <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
       </div>
 
+      <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
+        <h2 class="govuk-heading-l">How we work</h2>
```

Comment:
Can we look at another heading? It overlaps with the 'How we work' pages of the community section.

Reply:
Principles and strategies we use